packages/components/src/internal/components/navigation/ProductMenu.tsx

```diff
@@ -279,11 +282,14 @@ interface ProductMenuButtonTitle {
     routes: any[];
 }
 
+const HOME_PATH = '/home';
+const HOME_TITLE = 'Home Project';
```

Comment:
You could export these constants to be used in tests.

Reply:
done